functional_tests/steps/bundles.py

```diff
@@ -121,3 +129,337 @@ def the_user_can_see_the_bundle_details_with_creator(context: Context) -> None:
+
+
+@given("the {user_role} is a member of the preview team")
+def add_user_to_preview_teams(context: Context, user_role: str) -> None:
+    user = context.users[user_role]["user"]
+    user.teams.add(context.team)
+
+
+@given("there are {number_of_bundles} bundles with {bundle_details}")
+def multiple_bundles_create(context: Context, number_of_bundles: str, bundle_details: str) -> None:
+    bundle_dets = json.loads(bundle_details)
+    if bundle_dets:
+        context.bundles = []
+        context.bundlepages = []
+        context.bundleteams = []
+
+        for __ in range(int(number_of_bundles)):
+            bundle = BundleFactory()
+
+            if not hasattr(context.users, bundle_dets["creator_role"]) and bundle_dets["creator_role"] != "":
+                context.users[bundle_dets["creator_role"]] = create_user(bundle_dets["creator_role"])
+
+            bundle_status = BundleStatus.DRAFT
+            bundle_approved = False
+            if bundle_dets["status"] == "Approved":
+                bundle_status = BundleStatus.APPROVED
+                bundle_approved = True
+            elif bundle_dets["status"] == "In_Review":
+                bundle_status = BundleStatus.IN_REVIEW
+
+            if bundle_dets["creator_role"] != "":
+                user = context.users[bundle_dets["creator_role"]]["user"]
+
+                bundle.created_by = user
+                bundle.status = bundle_status
+                bundle.approved = bundle_approved
+
+            else:
+                bundle.status = bundle_status
+                bundle.approved = bundle_approved
+            bundle.save()
+
+            if bool(bundle_dets["preview_teams"]) and hasattr(context, "team"):
+                BundleTeam.objects.create(parent=bundle, team=context.team)
+                context.bundleteams.append({"parent": bundle, "team": context.team})
+            bundle.save()
+
+            if bool(bundle_dets["add_rel_cal"]) and hasattr(context, "release_calendar_page"):
+                page = context.release_calendar_page
+
+                bundle.release_calendar_page = context.release_calendar_page
+                BundlePage.objects.create(parent=bundle, page=page)
+                context.bundlepages.append(BundlePageFactory(parent=bundle, page=page))
+            else:
+                now = (datetime.now() + timedelta(hours=4)).strftime("%Y-%m-%d %H:%M")
+                bundle.scheduled_date = now
+            bundle.save()
+
+            if bool(bundle_dets["add_stat_page"]) and hasattr(context, "statistical_article_page"):
+                page = context.statistical_article_page
+                BundlePage.objects.create(parent=bundle, page=page)
+                context.bundlepages.append(BundlePageFactory(parent=bundle, page=page))
+            bundle.save()
+            context.bundles.append(bundle)
+
+        sorted(context.bundles, key=lambda x: x.name)
```

Comment:
The `sorted()` function _returns_ a sorted list.

https://docs.python.org/3/library/functions.html#sorted